experimental/README.md

```diff
@@ -11,7 +11,7 @@ please feel free to provide any feedback on these features you wish.
 
 ## Install Docker experimental
 
-Unlike the regular Docker binary, the experimental channels is built and
+Unlike the regular Docker package, the experimental package is built and
 updated nightly on https://experimental.docker.com. From one day to the
```

Comment:
Also tweaking of wording `s/From one day to the next/From one release to the next/`